### PR TITLE
nerffs glass breakdown

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
nerffs glassification

## Why It's Good For The Game
glass is annoying for a lot of players and punishing people with good sanity, this sould make it less. .. "bad"

## Changelog
:cl:Fernandos33
tweak: glass breakdown now don't last 5mins anymore. now it lasts only 2. and happens 3times only in 20 secs intervals with 1 sec duration.
/:cl:

### CODE REVIEW

**Comment by SpadesSlick** [comment]
Please make changelog entry that describes changes properly. This "glass now don't last as long" is rather incoherent.

File: code/modules/sanity/breakdowns.dm [hunk 1]
```diff
@@ -391,11 +391,11 @@
 
 /datum/breakdown/negative/glassification
 	name = "Glassification"
-	duration = 5 MINUTES
+	duration = 1 MINUTES //last less
```
Comment (SpadesSlick) [inline code/modules/sanity/breakdowns.dm:394]: Please don't leave comments that don't make sense outside of the review.

File: code/modules/sanity/breakdowns.dm [hunk 2]
```diff
@@ -391,11 +391,11 @@
...
 	restore_sanity_post = 40
 	var/time
-	var/cooldown = 15 SECONDS
-	var/time_view = 5 SECONDS
+	var/cooldown = 20 SECONDS // 3 sounds less bad
```
Comment (SpadesSlick) [inline code/modules/sanity/breakdowns.dm:397]: Same here.

File: code/modules/sanity/breakdowns.dm (hunk 1 above)
Comment (SpadesSlick) [reply code/modules/sanity/breakdowns.dm:394]: Also, try 2 minutes firts.

File: code/modules/sanity/breakdowns.dm (hunk 2 above)
Comment (fernandos13) [reply code/modules/sanity/breakdowns.dm:397]: roger that

**Pull request merged**